vtex/faststore
# chore: Removes `Form` component

## What's the purpose of this pull request?

We have decided to remove this component since keeping it has no benefits.

## How to test it?

Run `yarn dev` on `faststore/core` and check if `Newsletter` component it's working as before.
